echopype/echodata/echodata.py

```diff
@@ -93,6 +92,8 @@ def cleanup_swap_files(self):
                         v.store for k, v in dask_graph.items() if "original-from-zarr" in k
                     ]
                     fs = zarr_stores[0].fs
+                    from ..utils.io import delete_zarr_store
+
```

Comment:
Just curious why you decided to move these imports to within the function? Is it because they are only used once?

Reply:
@leewujung they were causing cyclical import and also the reason you've mentioned since anyway they're being used only once, I don't think we need to define them globally in the file.